libraries/ESP8266WiFi/src/include/ClientContext.h

```diff
@@ -345,16 +346,21 @@ class ClientContext
         if (_pcb->snd_queuelen >= TCP_SND_QUEUELEN) {
             can_send = 0;
         }
-        size_t will_send = (can_send < left) ? can_send : left;
-        if (will_send) {
-            const uint8_t* buf = _datasource->get_buffer(will_send);
-            err_t err = tcp_write(_pcb, buf, will_send, TCP_WRITE_FLAG_COPY);
-            _datasource->release_buffer(buf, will_send);
+		size_t will_send = (can_send < left) ? can_send : left;
```

Comment:
These lines seem to use TABs for indentation, but the original file uses spaces. Could you please reformat to match indentation of the surrounding code?